Expensify/App
# Allow users to log in with sms logins without appending "@expensify.com"

### Details

- Adds normalizeLogin to User.js (follow the [precedent](https://github.com/Expensify/Web-Expensify/blob/74e3356e0b18aaa20771a7ed89cfbbe30d647648/site/lib/user.js#L1124-L1143) set in Web)
- Normalizes the user's login before calling API.Authenticate in Session.js 
- Removes a stray `@expensify.sms` from the Settings page. 

### Fixed Issues
<Please replace GH_LINK with the link to the GitHub issue this Pull Request is fixing>
Fixes Expensify/Expensify/issues/153413

### Tests


1. Log into E.cash with an account that has an SMS number as the primary login 
2. Enter your password and confirm you're able to successfully log in without adding "@expensify.sms" to your login. 
2. Navigate to the settings page and confirm you don't see `@expensify.sms` in the login displayed under your display name. 

### Tested On


### Screenshots


**OLD**
![image](https://user-images.githubusercontent.com/16747822/106836802-aebc7280-664e-11eb-82bb-1acbf851be0a.png)


#### Web
<img width="600" alt="" src="https://user-images.githubusercontent.com/16747822/106975245-03bdbe80-670b-11eb-8602-cdf6ce98efa1.png">

#### Mobile Web
<img width="300" alt="" src="https://user-images.githubusercontent.com/16747822/106975257-0a4c3600-670b-11eb-8494-dc22ce8f492b.png">

#### Desktop
<img width="600" alt="" src="https://user-images.githubusercontent.com/16747822/106975342-3962a780-670b-11eb-9bfd-468767cef903.png">

#### iOS
<img width="300" alt="" src="https://user-images.githubusercontent.com/16747822/106975371-467f9680-670b-11eb-800a-0d7f81a0adc0.png">

#### Android
<img width="300" alt="" src="https://user-images.githubusercontent.com/16747822/106975406-54cdb280-670b-11eb-904b-d4c676317ab5.png">
